src/hmem_cuda.c

```diff
@@ -102,6 +102,12 @@ CUresult ofi_cuInit(unsigned int flags)
 
 int cuda_copy_to_dev(uint64_t device, void *dev, const void *host, size_t size)
 {
+	int err;
+
+	err = cuda_gdrcopy_to_dev(device, dev, host, size);
```

Comment:
Would it make sense to have the HMEM copy dispatch table call cuda_gdrcopy_to_dev() directly? Similar question for cuda_gdrcopy_from_dev().

Reply:
Good idea, will do that.